src/shared/components/button/index.js

```diff
@@ -6,9 +6,9 @@ import Link from 'shared/components/link';
 
 import styles from './index.module.css';
 
-const Button = ({ translationId, path, customClass, intl: { messages } }) => (
+const Button = ({ translationId, path, className, intl: { messages } }) => (
     <Link to={ path } className={ styles.link } >
-        <div className={ classNames(styles.customButton, customClass) }>
+        <div className={ classNames(styles.customButton, className) }>
             { messages[translationId].toUpperCase() }
```

Comment:
It is better and performant to do this type of string manipulations in CSS rather than JS.

```css
text-transform: uppercase
```